forth_src/compat.fs

```diff
@@ -53,6 +53,7 @@ begin over c@ digit? over and while
 1+ r> 1- repeat ;
 
 \ from FIG UK
+: ?negate 0< if negate then ;
```

Comment:
It is better to remove `?negate` IMHO. It is not a standard word.

Reply:
It's used by `SM/REM`, just below.